### PR TITLE
chore(functions): actually support env vars in target URL

### PR DESCRIPTION
**Pull request by ace-n**

### CODE REVIEW

File: functions/helloworld/test/sample.integration.http.test.js [hunk 1]
```diff
@@ -19,8 +19,8 @@ const path = require('path');
 const requestRetry = require('requestretry');
 const uuid = require('uuid');
 
-const PORT = 9010;
-const BASE_URL = `http://localhost:${PORT}`;
+const PORT = process.env.PORT || 8010;
```
Comment (grant) [inline functions/helloworld/test/sample.integration.http.test.js:22]: I prefer 8080 if we are going to change the PORT.

**Review by grant — APPROVED** [review]

File: functions/helloworld/test/sample.integration.http.test.js (hunk 1 above)
Comment (ace-n) [reply functions/helloworld/test/sample.integration.http.test.js:22]: The docs and other samples use `8010` - any objection to keeping it consistent?

Comment (grant) [reply functions/helloworld/test/sample.integration.http.test.js:22]: It looks like the docs only has Node right now:

https://cloud.google.com/functions/docs/testing/test-http#integration_tests

Let's use 8080, that is the default of the FF (makes it easier to run locally).

https://github.com/GoogleCloudPlatform/functions-framework#specification-summary

And the point of the sample isn't about customizing ports.

Comment (grant) [reply functions/helloworld/test/sample.integration.http.test.js:22]: Python uses 8080, I think Java might too.

Comment (ace-n) [reply functions/helloworld/test/sample.integration.http.test.js]: Got it. Changed this to `8080`.

**Comment by ace-n** [comment]
FYI - the environment variable multiplexing strategy I used (`sed`-ing in the correct value for the system tests) has been done before (in other DPE repos), but feels like a hack.

@fhinkel PTAL, and LMK if I should do something more fault-tolerant (maybe using different env var names for each? In the past, we wanted to point out the similarities between **integration** and **system** tests, but I don't think they're that similar anymore.)

**Review by fhinkel — APPROVED** [review]

**Pull request merged**